e2e/multi-cluster/multi_cluster_test.go

```diff
@@ -0,0 +1,101 @@
+package examples_test
+
+import (
+	"github.com/rancher/fleet/e2e/testenv"
+	"github.com/rancher/fleet/e2e/testenv/kubectl"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("multiCluster", func() {
+	var (
+		asset string
+		k     kubectl.Command
+		kd    kubectl.Command
+	)
+
+	BeforeEach(func() {
+		k = env.Kubectl.Context(env.Fleet).Namespace(env.Namespace)
+		kd = env.Kubectl.Context(env.Downstream)
+	})
+
+	JustBeforeEach(func() {
+		out, err := k.Apply("-f", testenv.AssetPath(asset))
+		Expect(err).ToNot(HaveOccurred(), out)
+	})
+
+	AfterEach(func() {
+		out, err := k.Delete("-f", testenv.AssetPath(asset))
+		Expect(err).ToNot(HaveOccurred(), out)
+
+	})
+
+	When("creating gitrepo resource", func() {
```

Comment:
Would it be possible to have an  `When("edit gitrepo resource"),`?

For example creating `multi-cluster/helm` (the one that contains a helm chart) Editing the fleet cluster label from `dev` to `test` or viceversa you'd end with different resources (pod numbers, iirc please check the `multi-cluster/helm`). I think in my opinion will complete the E2E check on CI but im not sure if that would require much effort in this framework.

Reply:
Yes, that would be a good future addition. That would check this https://github.com/rancher/fleet-examples/blob/master/multi-cluster/helm/fleet.yaml#L12-L14 functionality, right?

There is some work on another multi-cluster setup (standalone via terraform), I hope this will allow us to spin up three k3ds in total, so we can have two downstream clusters with different labels.